fiduswriter/document/static/js/modules/editor/index.js

```diff
@@ -553,7 +553,7 @@ export class Editor {
                         this.mod.footnotes.fnEditor.view.dispatch(footTr)
                     }
                 })
-                if (tr.steps) {
+                if (tr.steps.length) {
```

Comment:
not sure if this is a bug, but in my experience and based on Prosemirror types, tr.steps always exists. Let me know if you'd prefer to leave this change out. I didn't really see any place that is relying on the `updated` field

Reply:
Not sure, this may have been a bug in a previous PreseMirror version. Or it was just me imagining the steps being optional when they weren't.